core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -938,6 +941,97 @@ public Weighting createWeighting(HintsMap hintsMap, FlagEncoder encoder) {
         throw new IllegalArgumentException("weighting " + weighting + " not supported");
     }
 
+    private GHIntHashSet getBlockedEdges(HintsMap hints, FlagEncoder encoder) {
```

Comment:
Ok, I moved the common code to a different class, but the method is still quite long, not sure if we want to move everything out of the GraphHopper class?